web/packages/teleport/src/components/ServersideSearchPanel/useServerSideSearchPanel.ts

```diff
@@ -55,7 +55,8 @@ export default function useServersideSearchPanel({
         searchString,
         params.sort,
         params.kinds,
-        isAdvancedSearch
+        isAdvancedSearch,
+        false
```

Comment:
comment on what this bool is? (this is a pattern i see in the backend and i find it helpful if i can only read the code and can't jump and no hints)

Reply:
oh... i think this is why the pinned query param got disappeared when i cleared search from the pinned resources view

Reply:
Yeah this should be `params.pinnedResourcesOnly` thank you!